Frameworks/AddressBook/ABMultiValue.mm

```diff
@@ -19,74 +19,108 @@
 #import <StubReturn.h>
 #import "AssertARCEnabled.h"
 
+#import "ABMultiValueInternal.h"
+
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFTypeRef ABMultiValueCopyValueAtIndex(ABMultiValueRef multiValue, CFIndex index) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    if (multiValue == nullptr) {
+        return nullptr;
+    }
+
+    _ABMultiValue* multi = (__bridge _ABMultiValue*)multiValue;
```

Comment:
The __bridge cast is to allow the conversion from the C style pointer (ABMultiValueRef) into an objective-c pointer (without getting ARC involved -- there is also __bridge_transfer to allow ARC to take over, but the user is responsible for freeing the ABMultiValueRef)

---
In reply to: [75172038](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75172038) [](ancestors = 75172038,75171401)